src/js/components/SkipLinks/README.md

```diff
@@ -23,12 +23,103 @@ node
 
 Custom messages for SkipLinks. Used for accessibility by screen 
 readers. Defaults to `{
-  "skipTo": "Skip To"
+  "skipTo": "Skip To:"
 }`.
 
 ```
 {
   skipTo: string
 }
 ```
-  
\ No newline at end of file
+  
+## Theme
+  
+**skipLinks.position**
+
+Position of the layer content once opened. Expects `string`.
+
+Defaults to
+
+```
+top
+```
+
+**skipLinks.container**
+
+Any valid Box prop for the SkipLinks container. Expects `object`.
+
+Defaults to
+
+```
+{ elevation: 'large', pad: 'medium', round: 'small' }
+```
+
+**skipLinks.container.elevation**
+
+The container shadow. Expects `string`.
+
+Defaults to
+
+```
+large
+```
+
+**skipLinks.container.pad**
+
+The pad used for the layer container. Expects `string | object`.
+
+Defaults to
+
+```
+medium
+```
+
+**skipLinks.container.round**
+
+The rounding of the later container. Expects `boolean | string | object`.
+
+Defaults to
+
+```
+small
+```
+
+**skipLinks.label**
+
+Any valid Text prop for the text message. Expects `object`.
+
+Defaults to
+
+```
+{ margin: 'small', size: 'medium' }
+```
+
+**skipLinks.text.margin**
```

Comment:
`skipLinks.label.margin`?

Or, maybe I'm misunderstanding something. How does `skipLinks.label.margin` relate to `skipLinks.text.margin`?

Reply:
Oopsy, good catch.

Reply:
Fixed.